lib/client/api.go

```diff
@@ -3715,14 +3715,16 @@ func (tc *TeleportClient) applyProxySettings(proxySettings webclient.ProxySettin
 				proxySettings.DB.PostgresPublicAddr)
 		}
 		tc.PostgresProxyAddr = net.JoinHostPort(addr.Host(), strconv.Itoa(addr.Port(tc.WebProxyPort())))
-	case proxySettings.DB.PostgresListenAddr != "":
+		// When in TLS routing mode the web port should be used if no Postgres Public Addr is set
```

Comment:
Postgres public addr or listen addr? The comment says one thing and the code says another.

Reply:
Updated comment